contributors/guide/owners.md

```diff
@@ -115,6 +115,24 @@ emeritus_approvers:
 - emily    # 2019-01-05
 ```
 
+#### Cleanup
+
+In addition to the Emeritus process above, from time to time, it is necessary
+to prune inactive members from OWNERS files. A core principle in maintaining a
+healthy community is encouraging active participation. OWNERS have a higher
+activity requirement, as they directly impact the ability of others to contribute.
+If an owner should become inactive, they will be removed but may be added back
+should they become active again.
+
+
+An inactive OWNER is defined as someone with less than 10 Devstats recorded
+contributions within the past year, as shown by this [dashboard]. This is a
+conservative metric but should ensure only the removal of the most inactive
+OWNERS.
+- PR comments are less than 10 and Devstats count is less than 10 for a year
```

Comment:
So reading this I'm a bit confused - is an inactive OWNER moved to `emeritus_approvers`?  Or simply removed?  Looks like the automation moves them to emeritus?  If that's the case, then we should describe it with a sentence in this section ("if you are made emeritus here's where it goes")

Also, in the list of "valid fields" described for `OWNERS` here https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners we need to list emeritus approvers and link below.  I read the doc while reviewing felt that would be the right place to introduce emeritus (since now emeritus is more prevalent due to automation, and the likelihood someone wants to go back to approvers by contributing again).

Reply:
Also, I feel like the OWNERS file description should probably split "fields that list people" and "all other fields (i.e. `options` / `labels`)".  It's a key part of the process, and we shouldn't miss the opportunity to explain.  I can try to apply some feedback after the other stuff matures.

Reply:
fixing in https://github.com/kubernetes/community/pull/6339